src/quadpype/hosts/blender/plugins/publish/extract_camera_fbx.py

```diff
@@ -97,3 +95,17 @@ def process(self, instance):
 
         self.log.info("Extracted instance '%s' to: %s",
                       instance.name, representation)
+
+    @staticmethod
+    def get_asset_children(asset):
+        return list(asset.objects) if isinstance(asset, bpy.types.Collection) else list(list(asset.children))
```

Comment:
same question as for the extract camera abc : why `list(list(asset.children))` list of list for the children ?